pkg/schema/schema_test.go

```diff
@@ -1,4 +1,108 @@
 package schema
 
-// TODO tests for this stuff! Everything in this package is totally generic,
-// nothing is specific to Grafana
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"cuelang.org/go/cue"
+	"github.com/google/go-cmp/cmp"
+	"golang.org/x/tools/txtar"
+)
+
+var CasesDir = filepath.Join("load", "testdata", "artifacts", "dashboards", "trimdefault")
```

Comment:
I'd intended the `artifacts` subdirectory here to really be just plain, simple JSON files (not that that's well-documented anywhere). There's no need to reuse that dir for these cases, especially given that they're txtars. And doubly so because reaching into a different package's testdata directory isn't the greatest practice. `testdata/trimdefault` should be fine.